selfdrive/car/hyundai/carstate.py

```diff
@@ -146,7 +146,7 @@ def update(self, cp, cp_cam):
     if not self.CP.openpilotLongitudinalControl:
       aeb_src = "FCA11" if self.CP.flags & HyundaiFlags.USE_FCA.value else "SCC12"
       aeb_sig = "FCA_CmdAct" if self.CP.flags & HyundaiFlags.USE_FCA.value else "AEB_CmdAct"
-      aeb_warning = cp_cruise.vl[aeb_src]["CF_VSM_Warn"] != 0
+      aeb_warning = cp_cruise.vl[aeb_src]["CF_VSM_Warn"] != 0 or (cp_cruise.vl["SCC12"]["ACCMode"] == 1 and cp_cruise.vl["SCC12"]["TakeOverReq"] == 1)
       aeb_braking = cp_cruise.vl[aeb_src]["CF_VSM_DecCmdAct"] != 0 or cp_cruise.vl[aeb_src][aeb_sig] != 0
```

Comment:
```suggestion
      aeb_warning = cp_cruise.vl[aeb_src]["CF_VSM_Warn"] != 0
      aeb_braking = cp_cruise.vl[aeb_src]["CF_VSM_DecCmdAct"] != 0 or cp_cruise.vl[aeb_src][aeb_sig] != 0 or (cp_cruise.vl["SCC12"]["ACCMode"] == 1 and cp_cruise.vl["SCC12"]["TakeOverReq"] == 1)
```

Even though there was a warning on the cluster, we should have this signal under `aeb_braking` as the car was actively braking (-4 m/s^2 was actively being sent to the car) along with the "take over immediately" message being shown on the car's cluster.

Reply:
-4 m/s^2 is allowed by ISO 15622:2018 under 5 m/s (the speed here was 4.7 m/s^2), so it's more likely this was all SCC.